packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -87,6 +103,13 @@
     sa.PrimaryKeyConstraint("key", "version", name="services_meta_data_pk"),
 )
 
+
+#
+# services_access_rights table:
+#   Defines access rights (execute_access, write_access) on a service (key)
```

Comment:
as always... I wonder if these kind of comments are very useful. I mean ```services_access_rights``` is a pretty descriptive name. why the need to add yet another comment on top?

Reply:
I was documenting while reading the code and organizing  my thoughts but I admit it is unnecessary verbose.  Will remove.